test/distributions/test_transforms.py

```diff
@@ -552,6 +552,9 @@ def test_save_load_transform():
     stream = io.BytesIO()
     torch.save(dist, stream)
     stream.seek(0)
+    with torch.serialization.safe_globals(
+        [TransformedDistribution, AffineTransform, Normal]
+    ):
```

Comment:
Doesn't the following line need to be indented?

Reply:
done, thanks

Reply:
@mikaylagawarecki  please review this PR，thanks.